cvat-core/src/annotations-collection.ts

```diff
@@ -999,6 +999,12 @@ export default class Collection {
                     );
                 }
 
+                if (state.shapeType === 'mask') {
+                    if (state.points.length === 5) {
```

Comment:
maybe correct comparison is length < 6?

xtl, ytl, xbr, ybr and at least one filled pixel would have length 6

Reply:
Also two nested condition may be joined with &&

Reply:
This check is useful anyway
But I believe we should fix also cvat-canvas

Reply:
Fixed if case.
Could you clarify please what should we fix in cvat-canvas?

Reply:
I suppose a case when an empty mask created (using polygon minus tool)